2010-06-13  Aurelien Jarno  <aurelien@aurel32.net>

	* posix/group_member.c(__group_member): Correct off-by-one
	error.

diff --git a/posix/group_member.c b/posix/group_member.c
index 7bd9c46..104a055 100644
--- a/posix/group_member.c
+++ b/posix/group_member.c
@@ -41,8 +41,8 @@ __group_member (gid)
       size *= 2;
     } while (n == size / 2);
 
-  while (n >= 0)
-    if (groups[n--] == gid)
+  while (n > 0)
+    if (groups[--n] == gid)
       return 1;
 
   return 0;
